components/es-sidecar-service/pkg/server/purge.go

```diff
@@ -39,34 +42,33 @@ func (server *EsSidecarServer) PurgeTimeSeriesIndicesByAge(ctx context.Context,
 // PurgeDocumentsFromIndexByAge deletes all documents containing a date-mapped field "end_time" older than
 // in.OlderThanDays from in.Index.
 func (server *EsSidecarServer) PurgeDocumentsFromIndexByAge(ctx context.Context, in *api.PurgeRequest) (*api.PurgeResponse, error) {
-	logFields := log.Fields{
+	logctx := log.WithFields(log.Fields{
 		"id":                     in.GetId(),
 		"index":                  in.GetIndex(),
 		"delete-older-than-days": in.GetOlderThanDays(),
 		"custom-purge-field":     in.GetCustomPurgeField(),
-	}
+	})
 
-	log.WithFields(logFields).Debug("Initiating document purge from index")
+	logctx.Debug("Initiating document purge from index")
```

Comment:
Debug or Info? (I don't know how often we expect the message to appear.)

Reply:
I think Debug is fine here because we'll always log a success or failure later on